### PR TITLE
extsearch plugin: update ImmortalSeed

### PR DESCRIPTION
**Pull request by koblack**
- updated categories
- use selectedcats2 to select more than one category
- sort results by seeders
- fixed regex (first result has always category 'All Nuked')
- use shorter translation (leave more space to search query)

Screenshot_2019-08-13 ruTorrent v3 9(2) Screenshot_2019-08-13 ruTorrent v3 9(3)

### CODE REVIEW

**Comment by koblack** [comment]
@Novik  any comments?

File: plugins/extsearch/init.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ theSearchEngines.set = function(val, noSave)
 theSearchEngines.getEngName = function(eng)
 {
 	return(eng=="all" ? theUILang.All : 
-		eng=="public" ? theUILang.extAllPublic :
```
[the remark this answers is not in the document]
Comment (koblack) [reply plugins/extsearch/init.js]: Use shorter translation and leave more space to search query. If you don't like this idea, I can delete this part.

[the remark this answers is not in the document]
Comment (Novik) [reply plugins/extsearch/init.js]: Delete this, please.

**Comment by koblack** [comment]
PR updated

**Pull request merged**